mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -958,6 +983,7 @@ void CLeaderboardsTimes::ApplySchemeSettings(IScheme* pScheme)
     m_cFirstPlace = pScheme->GetColor("FirstPlace", Color(240, 210, 147, 50));
     m_cSecondPlace = pScheme->GetColor("SecondPlace", Color(175, 175, 175, 50));
     m_cThirdPlace = pScheme->GetColor("ThirdPlace", Color(205, 127, 50, 50));
+    m_cItemDefault = pScheme->GetColor("Panel.BgColor", Color(54, 54, 54, 255));
```

Comment:
The only worry I have with this is this BG color can go out of sync with this panel's bg color if it gets overridden. It might be best to define this also in the res file for this panel at least, and to default it to `GetBgColor()` in apply scheme settings.

Reply:
This is no longer an issue, the colors will no longer be re-set constantly